File: docs/dev_docs/nicepay/NICEPAY-ANDROID/NICEPAY-ANDROID/java/NicePayAppSample/app/src/main/java/sample/nicepay/co/kr/nicepayappsample/MainActivity.java
package sample.nicepay.co.kr.nicepayappsample;

import android.app.Activity;
import android.content.Intent;
import android.os.Bundle;
import android.view.View;
import android.widget.TextView;

public class MainActivity extends Activity {

    @Override
    protected void onCreate(Bundle savedInstanceState) {
        super.onCreate(savedInstanceState);
        setContentView(R.layout.activity_main);

        TextView verno = (TextView)findViewById(R.id.text_verno);
        if( verno != null ) {
            verno.setText("Ver " + BuildConfig.VERSION_NAME);
        }
    }

    public void actionPay(View v) {
        Intent intent = new Intent(MainActivity.this, WebViewActivity.class);
        if( intent != null ) {
            startActivity(intent);
        }
    }
}

File: docs/dev_docs/nicepay/NICEPAY-ANDROID/NICEPAY-ANDROID/kotlin/NicePayAppSample/app/src/main/java/kr/co/nicepay/nicepayappsample/MainActivity.kt
package kr.co.nicepay.nicepayappsample

import android.content.Intent
import android.os.Bundle
import android.view.View
import androidx.appcompat.app.AppCompatActivity
import kotlinx.android.synthetic.main.activity_main.*


class MainActivity : AppCompatActivity() {

    override fun onCreate(savedInstanceState: Bundle?) {
        super.onCreate(savedInstanceState)
        setContentView(R.layout.activity_main)

        text_verno.text = "Ver " + BuildConfig.VERSION_NAME

        btn_pay.setOnClickListener {
            val intent = Intent(this, WebViewActivity::class.java)
            startActivity(intent)
        }
    }
}
